tests/data/dialog_sample.txt

```diff
@@ -1,1000 +1,1000 @@
-Look, I haven't really focused on that kind of thing.    What's your cut?  How much?
-What's your cut?  How much?    I would say...half.  Half is fair.
-I would say...half.  Half is fair.    No.  No way.
-No.  No way.    But it's...
-But it's...    Thirty-percent.  No more.  Or I call another lawyer.  This is the biggest case of your life.  Don't try to negotiate.  Thirty percent.  Say yes or no.
-Thirty-percent.  No more.  Or I call another lawyer.  This is the biggest case of your life.  Don't try to negotiate.  Thirty percent.  Say yes or no.    This is not about money, Emil.  I need your trust in me.
-This is not about money, Emil.  I need your trust in me.    What else do you need?
-What else do you need?    I need to know about your background.  I need to know about your upbringing.  Why you're here.
-I need to know about your background.  I need to know about your upbringing.  Why you're here.    Give me another one, please.
-Tell me about yourself.  What you did as a young boy... what your parents were like.    My father always degraded me.  Killed my self-esteem.  And my mother was blind.
-My father always degraded me.  Killed my self-esteem.  And my mother was blind.    Your mother was blind?
-Your mother was blind?    Yeah, she went blind giving birth to me. She went to fucking black market doctor to induce me.
-Yeah, she went blind giving birth to me. She went to fucking black market doctor to induce me.    Back in the Czech Republic?
-Back in the Czech Republic?    Yeah, yeah...bad doctor gave her bad drugs which made her go blind.  And my father blamed me for her blindness...
-Yeah, yeah...bad doctor gave her bad drugs which made her go blind.  And my father blamed me for her blindness...    Your father blamed you for your mother's blindness?
-Your father blamed you for your mother's blindness?    Yeah, he hated me from day when I was born.  Put it out.  Can you put the cigarette out?
-That's what he did to me.  He put cigarettes out on me.    Your father put cigarettes out on you?
-Your father put cigarettes out on you?    Out on my back when I was a small boy.
-Out on my back when I was a small boy.    Can I see your back?
-I'm abused.  Don't you think?    I don't think it's abuse, I think it's torture.
-...so we kill someone famous and if we are caught, we are sent to mental hospital...    Officers, there's your killer, do your duty, arrest him!
-...my little sister and I shared a flat - I came home one night and a man was raping her.  His gun was on the chair... He came at me and I shot him.    Alright.  That's a justifiable homicide.
-Alright.  That's a justifiable homicide.    Yes, but he was a cop.
-Now I become custody of police department?    If you cooperate with the DA - maybe they'll help you with your situation.
-If you cooperate with the DA - maybe they'll help you with your situation.    I will if they don't send me back.
-I will if they don't send me back.  
```

Comment:
Ok, so what actually happened with that file....? Something with spacing/tabs?

Reply:
yep, the tabs went missing, i added them back